README.md

```diff
@@ -2,9 +2,11 @@
 # Infrastructure Management
 
 ## Deploying & provisioning with tape
-**Use Unbuntu precise64 (12.04 x64)** 
+**Use Ubuntu Server 14.04 LTS x64**
 
-**Enable ssh access via root user**
+**Provision with [packer](https://packer.io/)**
+
+```packer build -var 'aws_access_key=XXXXX' -var 'aws_secret_key=XXXXX' -var 'do_api_key=XXXXX' -var 'do_client_id=XXXXX' packer.json```
```

Comment:
You're right. It should be `do_api_token` for the latest authentication method, but I get validation errors from packer. Not sure if this is fixed but I don't use DO for any projects so I don't have much time to test it.